.github/workflows/CreatePackages.yml

```diff
@@ -0,0 +1,231 @@
+        path: |
+          artifact/VSIX/*.*
+       
+  test:
+    name: Test core templates
+    runs-on: ubuntu-latest
+    needs: [getVersion, build]
+    steps:
+      - uses: actions/checkout@v2
+      
+      - name: GetUpdatedBrowser
+        run: |
+          sudo apt-get update
+          Write-Host "Installing/Updating Chrome"
+          sudo apt-get --only-upgrade install google-chrome-stable
+        shell: pwsh
+      
+      - uses: actions/download-artifact@v2
+        with:
+          name: 'nuget'
+          path: ${{ github.workspace }}/nuget
+
+      - name: Run Core Tests
+        shell: pwsh
+        run: |
+              dotnet new -i "${{github.workspace}}/nuget/Magenic.Maqs.Templates.${{ needs.getVersion.outputs.buildNumber }}.nupkg"
+              mkdir Results
+              mkdir Appium
+              cd Appium
+              dotnet new Maqs.Appium
+              dotnet test --logger trx --results-directory "${{github.workspace}}/Results/"
+              cd ..
+              mkdir Base
+              cd Base
+              dotnet new Maqs.Base
+              dotnet test --logger trx --results-directory "${{github.workspace}}/Results/"
+              cd ..
+              mkdir Composite
+              cd Composite
+              dotnet new Maqs.Composite
+              dotnet test --logger trx --results-directory '${{github.workspace}}/Results/' -- 'TestRunParameters.Parameter(name=\"SeleniumMaqs:Browser\",value=\"HeadlessChrome\")' 
+              cd ..
+              mkdir Database
+              cd Database
+              dotnet new Maqs.Database
+              dotnet test --logger trx --results-directory "${{github.workspace}}/Results/"
+              cd ..
+              mkdir Email
+              cd Email
+              dotnet new Maqs.Email
+              dotnet test --logger trx --results-directory "${{github.workspace}}/Results/"
+              cd ..
+              mkdir Selenium
+              cd Selenium
+              dotnet new Maqs.Selenium
+              dotnet test --logger trx --results-directory '${{github.workspace}}/Results/' -- 'TestRunParameters.Parameter(name=\"SeleniumMaqs:Browser\",value=\"HeadlessChrome\")' 
+              cd ..
+              mkdir Webservice
+              cd Webservice
+              dotnet new Maqs.Webservice
+              dotnet test --logger trx --results-directory "${{github.workspace}}/Results/"
+              
+      - name: Test Report
+        uses: dorny/test-reporter@v1
+        with:
+          name: ReportResults
+          path: ${{ github.workspace }}/Results/**/*.trx    # Path to test results
+          reporter: dotnet-trx
+
+  
+  MakeNewRelease:
+    name: Create new draft release - v${{ needs.getVersion.outputs.buildNumber }}
+    needs: [getVersion, test]
+    runs-on: ubuntu-latest
+    environment: release
+
+    steps:      
+    - name: Create draft release v${{ needs.getVersion.outputs.buildNumber }}
+      shell: pwsh
+      run: |
```

Comment:
Did you see this [GH API action](https://github.com/marketplace/actions/github-script) or does it not fit your needs?